tests/unattended/install/pytest.ini

```diff
@@ -6,4 +6,5 @@ markers =
     wazuh_cluster: test for wazuh-clusterd it is meant to be executed on the master node if a wazuh cluster is configured.
     wazuh_worker: test for wazuh cluster worker nodes. It is meant to be executed on the worker nodes.
     elastic: tests to be executed on Elastic hosts.
```

Comment:
This is AIO?

Reply:
`elastic` is used for every ES host independently of the installation type.